stanza/utils/visualization/document_to_viz.py

```
@@ -0,0 +1,75 @@
+from stanza.models.common.constant import is_right_to_left
+import stanza
+import spacy
```

one request i have - currently we do not have spacy as a dependency, so it would be good to either hide this inside the `visualize_doc` call (it is not too expensive to import something multiple times, so it won't be wasteful to repeatedly call this) or wrap this in a try/except block to hide errors.  that way, spacy can be optional rather than required